scylla/benches/benchmark.rs

```diff
@@ -1,7 +1,11 @@
 use criterion::{criterion_group, criterion_main, Criterion};
 
 use bytes::BytesMut;
-use scylla::frame::types;
+use scylla::{
+    frame::types,
+    frame::value::ValueList,
+    transport::partitioner::{calculate_token_for_partition_key, Murmur3Partitioner},
+};
```

Comment:
Nit: putting the commit with the benchmark at the beginning of the PR should make it easier to use it to compare performance before and after the changes.